app/src/main/java/cm/aptoide/pt/app/view/AppViewPresenter.java

```diff
@@ -1073,7 +1075,7 @@ private void handleInstallButtonClick() {
                   completable = appViewManager.getAppModel()
                       .flatMapCompletable(appModel -> appViewManager.getAdsVisibilityStatus()
                           .flatMapCompletable(
-                              status -> downloadApp(action, appModel, status).observeOn(
+                              status -> downloadApp(action, appModel, status, false).observeOn(
```

Comment:
Sending false here, while few lines bellow, for the click on install button it is checking the value.

Reply:
Btw, resume download method , downgrade, etc should also send the correct apkfy value instead of false